test/Microsoft.ApplicationInsights.AspNetCore.Tests/HostingDiagnosticListenerTest.cs

```diff
@@ -133,6 +137,35 @@ public void TestSdkVersionIsPopulatedByMiddleware(bool isAspNetCore2)
             Assert.Contains(SdkVersionTestUtils.VersionPrefix, requestTelemetry.Context.GetInternalContext().SdkVersion);
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void TestSdkVersionIsPopulatedByMiddleware(bool isAspNetCore2)
```

Comment:
leaving the name as before.. the name is clearly not indicative of what test is doing. Will address in separte PR.